layers/+lang/go/packages.el

```diff
@@ -96,3 +99,9 @@
       :defer t
       :init
       (push 'company-go company-backends-go-mode))))
+
+
+(defun go/init-ob-go ()
+  (use-package ob-go
+    :if (configuration-layer/layer-usedp 'org)
+    :ensure t))
```

Comment:
@syl20bnr You're right, the `:ensure t` is not needed, removing it in the next push to the branch.

P.S. Sorry for the delayed response!